test/utilities/lwcUtils.ts

```diff
@@ -32,7 +33,10 @@ export async function createLwc(name: string): Promise<void> {
   log('createLwc() - Select the default directory');
   // Select the default directory (press Enter/Return).
   await inputBox.confirm();
-  await pause(Duration.seconds(3));
+  await checkFileOpen(workbench,
+    name + '.js',
+    { timeout: Duration.seconds(5) }
+  )
```

Comment:
Great new validation!  Just a semicolon nit here 🙂 

```suggestion
  await checkFileOpen(workbench,
    name + '.js',
    { timeout: Duration.seconds(5) }
  );
```